src/pages/login/index.js

```diff
@@ -59,72 +86,108 @@ const Login = () => {
   const appId = process.env.REACT_APP_ID_FACEBOOK;
 
   return (
-    <section className="row">
-      <article className="form column left-column">
-        <form onSubmit={handleSubmit(onSubmit)} noValidate>
-          <div className="circles"> </div>
-          <h1 className="title">{t('login.title')}</h1>
-          <h5 className="subTitle">{t('login.subTitle')}</h5>
-          <p className="explanation">{t('login.explanation')}</p>
-          <label htmlFor="email">{t('login.labels.email')}</label>
-          <Input
-            register={register}
-            type="email"
-            name="email"
-            error={errors.email}
-            handleFocus={handleFocus}
-          />
+    <>
+      <ContactModal show={showModal} setShow={setShowModal}>
```

Comment:
Could we extract the `ContactModal` code to another component? Same for the other sections maybe. I think this login page is a little bit big, what do you think about that? 😄